Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
It also looks like `npm install grunt-cli` is completly unnecessary, because grunt and grunt-cli are already installed by yarn. Just remove the `system "npm", "install", "grunt-cli", *Language::Node.local_npm_install_args` line and everythign should work fine.